frontend/src/components/MyTextField/MyTextField.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import * as PropTypes from "prop-types";
+import { useField } from "formik";
+import TextField from "@material-ui/core/TextField";
+
+const MyTextField = ({ label, ...props }) => {
+  const [field, meta] = useField(props);
+  const errorText = meta.error && meta.touched ? meta.error : "";
+  return (
+    <TextField
+      {...field}
+      label={label}
+      helperText={errorText}
+      error={!!errorText}
```

Comment:
This is mostly preference but I would avoid doing things like this in javascript. You never know what and how can get evaluated and it will be hard to debug. Better to be verbose and check for example for length or being undefined/null/empty string.